access-token-management/src/AccessTokenManagement/Crypto.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Duende Software. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See LICENSE in the project root for license information.
+
+using System.Security.Cryptography;
+using System.Text;
+using Duende.IdentityModel;
+
+namespace Duende.AccessTokenManagement;
+
+internal class Crypto
+{
+    public static string HashData(string data)
+    {
+        using (var sha = SHA256.Create())
```

Comment:
Minor: can use using declaration so don't need the bracing. Can do in a follow up PR.

Reply:
was a copy / paste from a different place, but agreed.